package.json

```diff
@@ -25,7 +25,7 @@
   "peerDependencies": {
     "@angular/core": ">=4.3.6",
     "@angular/http": ">=4.3.6",
-    "@blackbaud/auth-client": "^2.7.0",
+    "@blackbaud/auth-client": "^2.17.0",
```

Comment:
This should be `^2.17.1` since that is the version that contains the new zone parsing logic.